engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/customtransformers/StandaloneCustomExtractor.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.engine.standalone.utils.customtransformers
+
+import cats.data.Validated.Valid
+import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.context.ContextTransformation
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.standalone.api.StandaloneCustomTransformer
+import pl.touk.nussknacker.engine.standalone.api.types.InterpreterType
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object StandaloneCustomExtractor extends CustomStreamTransformer {
+
+  @MethodToInvoke(returnType = classOf[Void])
```

Comment:
why returnType? it's deprecated, isn't it?

Reply:
Now i see that this deprecation is a little bit confusing. For now deprecated is providing here precise type, but still (just like in `ReturningType` approach) you must define is output var will be added or not - see usage of `hasNoReturn` in `UIObjectDefinition` and `DefinitionPreparer`. I can be more precise here and put some TODO in `hasNoReturn`. WDYT?

Reply:
:+1: